test/agenda/cancel.js

```diff
@@ -0,0 +1,104 @@
+import test from 'ava';
+import {beforeEach, afterEach} from '../helpers';
+
+test.beforeEach(beforeEach);
+test.afterEach.always(afterEach);
+
+test('returns nothing', t => {
+  const {agenda} = t.context;
+
+  t.is(agenda.cancel('jobA'), undefined);
```

Comment:
@simison Should this throw an error since the job doesn't exist?

Reply:
Good question!

I feel that attempting to cancel a job that didn't exist kinda _successfully_ fulfils the intention here, so it would not need to error? These are always tricky ones.

I remember seeing quite a few issues around [cancelling jobs](https://github.com/agenda/agenda/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20cancel).